src/Portfolio/Portfolio.scss

```diff
@@ -0,0 +1,11 @@
+.portfolio {
+  background-image: url(../../assets/Portfolio.jpg);
+  height: calc(100vh - 60px);
```

Comment:
I get the idea but i dont think this works. Your content goes slightly further than 100vh on desktop (on my screen anyway) and on mobile it doesnt work at all.

I would say just go with 100%, it's at least fully responsive and still looks pretty good imo